specification.md

```diff
@@ -4768,6 +4810,32 @@ States that are marked as `usedForCompensation` can define [error handling](#Wor
 (errors not explicitly handled),
 workflow execution should be stopped, which is the same behavior as when not using compensation as well. 
 
+### Workflow Versioning
```

Comment:
WDYT about separating out the bits about schemaVersion from workflow versions?   It seems odd that these are lumped together as one concept.

Reply:
+1 will update